### ISSUE TITLE
Preview of a slide in slides list

### ISSUE THREAD

**Issue by JithinKS97** [JithinKS97/dynamic-learning-app#15]
At present the index of the slide is shown in the slides list. It would be better if we could show a preview of the slide instead of index just like in Google slides.

**Comment by nickmcintyre** [comment]
Agreed this would be nice to have. Do you think including the title would be too cluttered?

**Comment by JithinKS97** [comment]
Presently we don't have title for each slide, but may be we can include. It would help users to quickly navigate to each slide. May be we can show it partially so that it won't be cluttered?

**Comment by nickmcintyre** [comment]
I like that, perhaps the first word or an acronym ("Simple Harmonic Motion" is previewed as "Simple" or "SHM" by default).

**Comment by JithinKS97** [comment]
Yeah, that can be done.
